pkg/cmd/release/view/view.go

```diff
@@ -181,6 +181,7 @@ func renderReleasePlain(w io.Writer, release *shared.Release) error {
 	fmt.Fprintf(w, "tag:\t%s\n", release.TagName)
 	fmt.Fprintf(w, "draft:\t%v\n", release.IsDraft)
 	fmt.Fprintf(w, "prerelease:\t%v\n", release.IsPrerelease)
+	fmt.Fprintf(w, "immutable:\t%v\n", release.IsImmutable)
```

Comment:
This is a breaking change. As a rule of thumb, `gh` tends to keep machine-friendly outputs backwards compatible because we don't know how many users use them in what way. I know the output is formatted in a way that a sensible code/script that reads it should check the field names, but you'd never know how they're actually being used.

Given the importance of `release` family commands, I can imagine doing this will probably raise legitimate complaints.

Reply:
With Babak going on holiday, @williammartin and I were discussing this and believe additive changes like this are not considered breaking.

There is likely discussion about whether non-tty is still considered machine readable versus `--json` usage.